src/effects/builtin/noisegen/noiseviewmodel.cpp

```diff
@@ -0,0 +1,76 @@
+/*
+ * Audacity: A Digital Audio Editor
+ */
+#include "noiseviewmodel.h"
+
+#include "global/translation.h"
+#include "log.h"
+
+using namespace au::effects;
+
+QVariantList NoiseViewModel::types() const
+{
+    static QVariantList items;
+    if (items.empty()) {
+        items.append(QVariantMap {
+            { "text", QString::fromStdString(muse::trc("White", "not a color, but 'white noise' having a uniform spectrum")) },
```

Comment:
muse::trc("White", "not a color, but 'white noise' having a uniform spectrum") 
-> 
muse::trc("effects", "White", "not a color, but 'white noise' having a uniform spectrum")